apps/webpack.js

```diff
@@ -20,7 +21,9 @@ var baseConfig = {
     "blockly": "this Blockly",
     "react": "var React",
     "react-dom": "var ReactDOM",
-    "jquery": "var $"
+    "jquery": "var $",
+    "radium": "var Radium",
+    "bindings": true
```

Comment:
See the provides option. When a bundle says it provides one of these, we remove it from the list of externals.